cfgov/legacy/views/housing_counselor.py

```diff
@@ -2,6 +2,7 @@
 import re
 import requests
 import sys
+from urlparse import urljoin
```

Comment:
Would you mind alphabetizing this with the other third-party imports below (under `django.views.generic`)?